com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/java/com/broadcom/lsp/cdi/EngineModule.java

```diff
@@ -32,9 +38,12 @@
   protected void configure() {
     bind(CobolLanguageEngine.class);
     bind(CobolPreprocessor.class).to(CobolPreprocessorImpl.class);
-    bind(CopybookAnalysis.class).to(CopybookParallelAnalysis.class);
-    install(new FactoryModuleBuilder().build(AnalyseCopybookTaskFactory.class));
     bind(CobolPreprocessor.class).to(CobolPreprocessorImpl.class);
-    bind(CobolSemanticParser.class).to(CobolSemanticParserImpl.class);
+    bind(GrammarPreprocessor.class).to(GrammarPreprocessorImpl.class);
```

Comment:
names of this classes makes me think you are speaking about antlr auto generated classes